starknet_py/utils/data_transformer/errors.py

```diff
@@ -0,0 +1,6 @@
+class CairoSerializerException(TypeError):
+    """Exception thrown by CairoSerializer"""
+
+
+class InvalidDataException(ValueError):
```

Comment:
I would add an abstract class to inherit from that would mean any error coming from data_transformer. This way protostar can expect this one error.